### ISSUE TITLE
Nothing in Rome folder

### ISSUE THREAD

**Issue by hartbit** [neonichu/Rome#12]
I've trying out Rome with this Podfile:

```
platform :osx, '10.10'
use_frameworks!

plugin 'cocoapods-rome'

pod 'XcodeEditor'
```

And here's my attempt at building:

```
$ pod install --no-integrate                                                                                                                                      ~/Projects/atipik-tools.iosapp/
Updating local specs repositories
Analyzing dependencies
Downloading dependencies
Using XcodeEditor (1.6.2)
Generating Pods project
Sending stats
Building frameworks
Built 0 frameworks
Copying frameworks to `Rome`
```

The Pods project in the Pods folder seems fine, but there's no Rome folder, and it printed `Built 0 frameworks`.

If it's any help, here's a link to a gist with the same command but in `--verbose` mode:

https://gist.github.com/hartbit/f06e919bb1320fa2f977

**Comment by neonichu** [comment]
Which versions of CP and Rome are you using?

**Comment by neonichu** [comment]
Probably fixed by 8658bfc7c53cf5a430a7cc5f1f2a56e901723f09
